src/components/mortgage-investment-compare/index.js

```diff
@@ -76,33 +112,30 @@ class MortgageInvestmentCompare extends React.Component {
 
   // Returns future value dynamically depending on mortgage term length.
   calculateFV (option1, option2, input) {
+    const investmentRate = this.state.includeROI
+      ? input.investmentRate / 100 / COMPOUND_FREQUENCY
+      : 0
+    const inflationRate = this.state.includeInflation
+      ? input.inflation / 100
+      : 0
+
```

Comment:
I pretty much do this same thing a little below in `getResult`. Not a fan of code duplication and tbh, we don't really need `calculateFV` anymore as we can just pull the final FV values from the amortization table, but I guess it's nice to have for now to double check our work. I'm sure we'll eventually get rid of the bulk calc.